### PR TITLE
feat: support separate maxHeights for facets and results tables

### PR DESCRIPTION
**Pull request by crfmc**
Corresponds to [Trello ticket 250](https://trello.com/c/pelxx3O1/250-smaht-embedded-table-scroll-functionality?filter=member:crfmc).
- Support separate maxHeights for facets and results tables

### CODE REVIEW

**Review by Bianca-Morris — APPROVED** [review]

File: src/components/browse/components/ControlsAndResults.js [hunk 1]
```diff
@@ -75,8 +75,10 @@ export class ControlsAndResults extends React.PureComponent {
             isOwnPage = true,         // <- False when rendered by EmbeddedSearchView, else is true when from a SearchView
             isContextLoading = false, // <- Only applicable for EmbeddedSearchView, passed in by VirtualHrefController only, else is false always since we initialize immediately over search-response context that already has first 25 results
 
-            // From EmbeddedSearchView/manual-entry, used if isOwnPage is true
-            maxHeight = SearchResultTable.defaultProps.maxHeight,
+            // From EmbeddedSearchView/manual-entry, used if isOwnPage is true,
+            maxHeight,
+            maxFacetsBodyHeight = SearchResultTable.defaultProps.maxHeight,
```
Comment (utku-ozturk) [inline src/components/browse/components/ControlsAndResults.js:80]: @crfmc  It looks like `SearchResultTable.defaultProps.maxHeight `does not exist anymore. Is it ok that both `maxFacetsBodyHeight` and `maxResultsBodyHeight` are set as undefined?

**Comment by utku-ozturk** [comment]
@crfmc I have tested it in 4DN locally. I think, we need to update static sections or pages that are using embedded search table with facets (we need to pass a prop?). For instance, when I run [this page](https://mastertest.4dnucleome.org/static-sections/4570c6b5-dad9-4f34-b366-9895b461334d/) in localhost, I am getting the client-side error below:

Screen Shot 2024-07-03 at 15 35 35

**Comment by utku-ozturk** [comment]
Update: Adding maxHeight to defaultProps looks fixed it.

File: src/components/browse/components/ControlsAndResults.js (hunk 1 above)
Comment (crfmc) [reply src/components/browse/components/ControlsAndResults.js:80]: Thank you for pointing this out! Yes I was running into the same error on CGAP, I am adding maxHeight back to defaultProps.

Comment (crfmc) [reply src/components/browse/components/ControlsAndResults.js:80]: Should work properly now on CGAP and Fourfront

**Review by utku-ozturk — APPROVED** [review]
Yeah, I've re-tested 0.1.86b3 locally, it looks good to me! thanks for the fix.

**Pull request merged**